docs/rapid-online/rapidplan-online-basics/geometries.md

```diff
@@ -0,0 +1,37 @@
+---
+
+sidebar_position: 5
+
+---
+# Geometries
+
+Geometries apply to the path either the edge of an object or the span of a line-based object takes between two control points.
+
+There are three geometry types:
+
+## Line
+
+Simply the shortest path between two points - straight.
+
+## Spline
+
+Spline applies a curve based on the **Tension** value you input in the object's **Properties**.
+
+-**Tension** values can be any value less than 1.
```

Comment:
This line will not be recognized as list item because 1 space symbol is missing between - and *.
![image](https://github.com/user-attachments/assets/2881f9f8-70a0-438a-8f62-d215f6f762c9)
This problem was highlighted by the style checker:
![image](https://github.com/user-attachments/assets/95d88a3e-cb9d-4233-b63a-c74cd144c88e)